README.md

```diff
@@ -225,6 +225,19 @@ If building code using the `System` global in Webpack, the following config is n
 }
 ```
 
+As of webpack@5.0.0-beta.15, you can access [SystemJS context](/docs/system-register.md#format-definition) via `__system_context__`:
+
+```js
+// Log the URL of the current SystemJS module
+// http://localhost:8080/dist/js/main.js
+console.log(__system_context__.meta.url);
```

Comment:
Can `import.meta.url` not be used directly?

Reply:
No - webpack hasn’t decided how it will handle import.meta yet.


https://github.com/webpack/webpack/issues/6719